### PR TITLE
ACM-9979: Remove or make a note about using hypershift CLI

### PR DESCRIPTION
**Pull request by rokej**
https://issues.redhat.com/browse/ACM-9979

There are still some references in the doc of using the "hypershift" CLI which is not downloadable by end-users. Remove or make a note about how to get it.

### CODE REVIEW

File: clusters/hosted_control_planes/deploy_aws_private_clusters.adoc [hunk 1]
```diff
@@ -106,6 +106,14 @@ To access a private cluster, you use a bastion.
 hypershift create bastion aws --aws-creds=$AWS_CREDS --infra-id=$INFRA_ID --region=$REGION --ssh-key-file=$SSH_KEY
 ----
 
+*Note:* `hypershift` CLI is not available for download. Use the following commands to extract it from the hypershift operator pod. Replace `operator-747645c946-27w5k` with the actual operator pod name in `hypershift` namespace.
```
Comment (xenolinux) [inline clusters/hosted_control_planes/deploy_aws_private_clusters.adoc]:
```suggestion
*Note:* The `hypershift` CLI is not available to download. Use the following commands to extract it by using the HyperShift Operator pod present in the `hypershift` namespace. Replace `<hypershift-operator-pod-name>` with your HyperShift Operator pod name.
```

File: clusters/hosted_control_planes/deploy_aws_private_clusters.adoc [hunk 2]
```diff
@@ -106,6 +106,14 @@ To access a private cluster, you use a bastion.
...
++
+----
+oc project hypershift
+oc rsync operator-747645c946-27w5k:/usr/bin/hypershift-no-cgo .
+mv hypershift-no-cgo hypershift
```
Comment (xenolinux) [inline clusters/hosted_control_planes/deploy_aws_private_clusters.adoc]:
```suggestion
oc project hypershift
oc rsync <hypershift-operator-pod-name>:/usr/bin/hypershift-no-cgo .
mv hypershift-no-cgo hypershift
```

File: clusters/hosted_control_planes/manage_aws_infra_iam.adoc [hunk 3]
```diff
@@ -488,6 +488,14 @@ hypershift create infra aws --name CLUSTER_NAME \ <1>
 <5> Replace `REGION` with the region where you want to create the infrastructure for your cluster.
 <6> Replace `OUTPUT_INFRA_FILE` with the name of the file where you want to store the IDs of the infrastructure in JSON format. You can use this file as input to the `hcp create cluster aws` command to populate fields in the `HostedCluster` and `NodePool` resouces.
 
+*Note:* `hypershift` CLI is not available for download. Use the following commands to extract it from the hypershift operator pod. Replace `operator-747645c946-27w5k` with the actual operator pod name in `hypershift` namespace.
```
Comment (xenolinux) [inline clusters/hosted_control_planes/manage_aws_infra_iam.adoc]:
```suggestion
*Note:* The `hypershift` CLI is not available to download. Use the following commands to extract it by using the HyperShift Operator pod present in the `hypershift` namespace. Replace `<hypershift-operator-pod-name>` with your HyperShift Operator pod name.
```

File: clusters/hosted_control_planes/manage_aws_infra_iam.adoc [hunk 4]
```diff
@@ -488,6 +488,14 @@ hypershift create infra aws --name CLUSTER_NAME \ <1>
...
++
+----
+oc project hypershift
+oc rsync operator-747645c946-27w5k:/usr/bin/hypershift-no-cgo .
+mv hypershift-no-cgo hypershift
```
Comment (xenolinux) [inline clusters/hosted_control_planes/manage_aws_infra_iam.adoc]:
```suggestion
oc project hypershift
oc rsync <hypershift-operator-pod-name>:/usr/bin/hypershift-no-cgo .
mv hypershift-no-cgo hypershift
```

Comment (xenolinux) [reply clusters/hosted_control_planes/manage_aws_infra_iam.adoc]: Same things to consider as before

**Review by xenolinux — COMMENTED** [review]
Commented a few considerations ; otherwise looks good to me!

**Comment by rokej** [comment]
@xenolinux Thank you for reviewing

**Review by xenolinux — APPROVED** [review]
LGTM!

**Pull request merged**